src/app/views/feedback/answers.jsx

```diff
@@ -17,46 +17,31 @@ import {
   DialogActions,
   Divider,
   Card,
-  // MenuItem,
-  // Input,
-  // FormControlLabel,
-  // Checkbox,
-  // Tooltip,
 } from '@material-ui/core';
 import { Link, useHistory } from 'react-router-dom';
 import dayjs from 'dayjs';
 
-import bc from 'app/services/breathecode';
+import bc from '../../services/breathecode';
 
-// import axios from '../../../axios';
 import { useQuery } from '../../hooks/useQuery';
 import { DownloadCsv } from '../../components/DownloadCsv';
 
 const relativeTime = require('dayjs/plugin/relativeTime');
 
 dayjs.extend(relativeTime);
 
-const stageColors = {
-  INACTIVE: 'bg-gray',
-  PREWORK: 'bg-secondary',
-  STARTED: 'text-white bg-warning',
-  FINAL_PROJECT: 'text-white bg-error',
-  ENDED: 'text-white bg-green',
-  DELETED: 'light-gray',
-};
-
 const Answers = () => {
+  const query = useQuery();
+  const history = useHistory();
+  const queryLike = query.get('like') || ''
   const [isAlive, setIsAlive] = useState(true);
   const [isLoading, setIsLoading] = useState(false);
   const [items, setItems] = useState({
     page: 0,
   });
   const [querys, setQuerys] = useState({});
-  const query = useQuery();
-  const history = useHistory();
   const [queryLimit, setQueryLimit] = useState(query.get('limit') || 10);
```

Comment:
same as before, use only one querys object instead of separate queryLike, queryLimit, etc.